src/components/TodosList/TodosList.tsx

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+
+interface TodoListProps {
+  todoList: Todo[] | undefined;
```

Comment:
Remove undefined type as even if you don't have a single todo todoList will be just empty array